### ISSUE TITLE
RPI 2 Missing views and options for button visibility in videoinfo 

### ISSUE THREAD

**Issue by vdb86** [marcelveldt/skin.titan#186]
Hi Marcel,
I updated my RPI 2 setup this morning and was trying to get Titan Jarvis beta on it.
Everything was going smoothly until I saw that I was missing views in Movie and TV show library.
I also noticed that no options were available when I click on "Select which buttons to show in videoinfo".

I tried rebooting, I also uninstalled the skin and removed it's settings and then copied it from my computer.
The whole thing was acting the same, the views were missing and the options.
I have never seen this before.
Basically these views are missing:
Panel
Panel details
Panel wall
Panel Low
Big Panel
Big Panel details
Big Panel Wall
Horizontal Panel

They are present in Forced views/Enable/disable views, just not available in libraries.
I'm running Kodi Jarvis stable on OSMC, the titan skin is 3.6.30.
Let me know if you need any more information or logs.

**Comment by vdb86** [comment]
Just tried Titan stable 3.6.25 and it has all the views that it should.
I'm completely confused.

**Comment by marcelveldt** [comment]
Hmm, must be a bug then, I'll look into it.
You are sure that you are looking at actual movies content ?

Panel views are not available for episodes

**Comment by vdb86** [comment]
Yes I'm looking at movies content.
I've tested the whole thing with installing Titan stable 3.6.25 and it works just fine.
Not sure why there's a difference between my laptop and RPI 2.

**Comment by vdb86** [comment]
Local trailer also won't play.
With stable release they work.

**Comment by marcelveldt** [comment]
works fine on my setup
any errors in the log ?

**Comment by vdb86** [comment]
That's weird, had two errors, but nothing to do with Titan.
Let me test tomorrow a completely fresh setup, we'll see how that one behaves.

**Comment by vdb86** [comment]
Btw, everything works fine on my tablet too...
Basically using tablet with the touch support you put in is awesome.
There are a couple of views or menus that are missing the navigation buttons, but you almost got all of them!
If needed I can help you hunt the rest down.

**Comment by marcelveldt** [comment]
OK, I will push a new beta in a few minutes to the repo, includes a whole bunch of fixes for some small bugs.

Yes, please let me know which windows/dialog still need the touch icons. I thought I had them all but I could have missed a few spots ;-)

**Comment by vdb86** [comment]
I'll make a separate issue and put them there!
Well since you have a ton of features, views and options I'm not sure if it would be possible for anyone to test everything :D

**Comment by vdb86** [comment]
Got to testing with a fresh setup and everything is working correctly.
If I find out what was causing this I'll let you know.
In the mean time I'll just close this issue since it's basically just me for some reason.
